### ISSUE TITLE
4.10 Beta deployment to Ubuntu 16.04 w/Core 2.1.3 bug?

### ISSUE THREAD

**Issue by VR-Architect** [nopSolutions/nopCommerce#3140]
nopCommerce version:  4.10 Beta

Steps to reproduce the problem: 

Do a Release/Mixed Platforms build with latest VS2017.

I copied everything under the presentation release folder to the .NET Core Ubuntu but received an error on run.

C:\nopCommerce_4.10_Beta_Source\Presentation\Nop.Web\bin\Release\netcoreapp2.1\

`Error:  An assembly specified in the application dependencies manifest (Nop.Web.deps.json) was not found:    package: 'AutoMapper.Extensions.Microsoft.DependencyInjection', version: '4.0.1'    path: 'lib/netstandard2.0/AutoMapper.Extensions.Microsoft.DependencyInjection.dll'`

Do we need to copy the release files under all the libraries too? What about Nop.Web.Framework and the plugins?Also note the folder path in the error message is netstandard2.0 and not 2.1. Is this correct?

BTW, here is the commands I used on the server if it helps anyone:  
```
sudo mkdir /var/www/nopCommerce  
sudo cp -a /home/developer/ftp/files/. /var/www/nopCommerce/      <-- Developer is the FTP user account  
sudo dotnet /var/www/nopCommerce/Nop.Web.dll
```

Thanks for the help.
--

**Comment by VR-Architect** [comment]
I manually copied that dll to the root folder of the project and reran the app. Now there is this error:

```
Error:
  An assembly specified in the application dependencies manifest (Nop.Web.deps.json) was not found:
    package: 'AutoMapper', version: '7.0.1'
    path: 'lib/netcoreapp2.0/AutoMapper.dll'
```

**Comment by VR-Architect** [comment]
SO move that file and then a new error. So I copied all the Nop.Core release folder contents and now this:

```
Error:
  An assembly specified in the application dependencies manifest (Nop.Web.deps.json) was not found:
    package: 'BundlerMinifier.Core', version: '2.8.391'
    path: 'lib/netcoreapp2.0/dotnet-bundle.dll'


```

**Comment by VR-Architect** [comment]
OK, this was my first time deploying a .NET Core app to Linux and figured it out. The issue was how I was publishing the project. Instead of just clicking the green button at the top to do a release build, I had to do an actual publish. 

To publish, I created a new directory on my developer laptop and published to it. This placed all dependency files needed. Then FTP those files over to my Linux server. Hope this helps others.
